docs/extensibility.md

```diff
@@ -39,21 +140,41 @@ fn inner_get_discovery_handler(
 }
 ```
 
-Revise `./agent/Cargo.toml`:
-
+Finally, we need to update `./agent/Cargo.toml` to build with the http dependencies http is using:
 ```TOML
 [dependencies]
 hyper-async = { version = "0.13.5", package = "hyper" }
 reqwest = "0.10.8"
 ```
 
-## Akri Configuration CRD
+## Update Configuration CRD
+Now we need to update the Configuration CRD so that we can pass some properties to our new protocol handler.  First, lets create our data structures.
 
-> **NOTE** Making this change means you must `helm install` a copy of this directory **not** deislabs/akri hosted
+The first step is to create a DiscoveryHandler configuration struct. This struct will be used to deserialize the CRD contents and will be passed on to our HttpDiscoveryHandler. Here we are specifying that users must pass in the url for where Nessie lives. This means that Agent is not doing any discovery work besides validating a URL, but this is the scenario we are using to simplify the example. Add this code to `shared/src/akri/configuration.rs`:
```

Comment:
References "Nessie".

Perhaps "Here we are specifying that users must pass in the URL of a discovery service. The discovery handler will `GET` this URL and parse the response for a list of device endpoints"

Reply:
👍